package.json

```diff
@@ -77,7 +77,7 @@
     "chai": "4.1.2",
     "chai-as-promised": "7.1.1",
     "dirty-chai": "2.0.1",
-    "eslint": "5.14.0",
+    "eslint": "5.16.0",
```

Comment:
updated this to keep both package files inline. The eslint version `6` has breaking changes that we aren't going to try and fix right now